entrypoint.sh

```diff
@@ -6,10 +6,12 @@ export BALENARC_BALENA_URL
 
 set -e
 
-[ -z "${BALENA_TOKEN}" ] && echo "BALENA_TOKEN input cannot be empty" && exit 1
+BALENA_API_KEY=${BALENA_API_KEY:=${BALENA_TOKEN:=}}
 
-# Authenticate CLI with token
-balena login --token "${BALENA_TOKEN}" 
+[ -z "${BALENA_API_KEY}" ] && echo "One of balena_api_key or balena_token inputs must not be empty" && exit 1
```

Comment:
Can you make this a github actions error message so it shows up in the annotations and is easy to spot: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message
```suggestion
[ -z "${BALENA_API_KEY}" ] && echo "::error::One of balena_api_key or balena_token inputs must not be empty" && exit 1
```

Reply:
Done [here](https://github.com/balena-io/deploy-to-balena-action/pull/214/files#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2R12)